src/components/App.js

```diff
@@ -0,0 +1,11 @@
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+const App = () => (
+  <div className="container">
+    <Display result="0" />
+    <ButtonPanel />
+  </div>
+);
```

Comment:
You should wrap all the child components in [React.Fragment](https://reactjs.org/docs/fragments.html#short-syntax) short syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/ReactFragment.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896092/))

Reply:
But it did compile, that's why I didn't use it.

Reply:
You're probably doing it the wrong way. Check my message on slack.